### PR TITLE
fix/ search autocomplete

### PR DESCRIPTION
**Pull request by San88Francisco**
https://app.clickup.com/t/8695jmjgm

### CODE REVIEW

File: src/components/RadioForm/BranchesNP.tsx [hunk 1]
```diff
@@ -77,19 +77,23 @@ export const NovaPoshtaBranch: FC<FormProps> = ({
...
     setValInputCity(value ?? '')
+
     clearErrors('cityBranches')
+
     if (CITY_DEFAULT.some((city) => city.name === value)) {
       setSettleRef(getDefaultCityRef(value, CITY_DEFAULT))
       clearErrors('cityBranches')
-    } else if (value && novaPoshtaCity.length > 0) {
+    } else if (value) {
+      console.log('good')
```
Comment (pqoqubbw) [inline src/components/RadioForm/BranchesNP.tsx]: remove console.log

File: src/hooks/useNovaPoshtaCity.ts [hunk 2]
```diff
@@ -33,16 +33,27 @@ const getCityRef = (
...
 
-  return selectedCity ? selectedCity.Ref : null
+    const selectedCity = novaPoshtaCity[0]?.Addresses.find(
+      (address) =>
+        address.MainDescription === mainDescription && address.Area === area
+    )
+
+    return selectedCity ? selectedCity.Ref : null
+  } else {
+    console.log('No match found')
```
Comment (pqoqubbw) [inline src/hooks/useNovaPoshtaCity.ts]: same here

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/components/RadioForm/BranchesNP.tsx (hunk 1 above)
Comment (San88Francisco) [reply src/components/RadioForm/BranchesNP.tsx]: done

File: src/hooks/useNovaPoshtaCity.ts (hunk 2 above)
Comment (San88Francisco) [reply src/hooks/useNovaPoshtaCity.ts]: done

**Comment by San88Francisco** [comment]
ready

**Pull request merged**